source/blog/2019-01-25-the-ember-times-issue-82.md

```diff
@@ -14,11 +14,11 @@ responsive: true
 
 ## [SECTION TITLE](#section-url)
 
-
 ---
 
-## [SECTION TITLE](#section-url)
+## [Bringing Angle Brackets to the Guides 🥳](https://github.com/ember-learn/guides-source/issues/139)
```

Comment:
Not sure if only on my laptop but the emoji is just a square for me?

Reply:
Thanks - it's the party "face" but maybe it's too new of an emoji, changed to the older :tada: